dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java

```diff
@@ -308,7 +308,7 @@ public void testWizardEndpoints() throws ApiException {
             BioWorkflow dsWorkflow = workflowsApi.addWorkflow(SourceControl.DOCKSTORE.name(), "foo", "bar");
             assertFalse("Should not reach this, should fail", false);
         } catch (ApiException ex) {
-            assertTrue("Should have error message that hosted workflows cannot be added this way.", ex.getMessage().contains("hosted workflows"));
+            assertTrue("Should have error message that hosted workflows cannot be added this way.", ex.getMessage().contains("User does not have access to the given source control registry"));
```

Comment:
Should re-use a string constant to avoid auto-breaking test when trivially changing the the error message

Reply:
(didn't notice that this wasn't a re-review request ... )